nubia/internal/interactive.py

```diff
@@ -113,7 +113,9 @@ def _build_cli(self):
         return cli
 
     def get_prompt_tokens(self, cli):
-        tokens = [(Token.Username, getpass.getuser())]
+        tokens = self._plugin.get_prompt_tokens()
```

Comment:
Please see the comment about `get_prompt_tokens` below.

Reply:
Hi @ylectric! I am having difficulty finding an example of a variable annotation in the project. What is your preferred form?

```
1. var = value # type: annotation
2. var: annotation; var = value
3. var: annotation = value
```

Reply:
We prefer the latter one.
I think the existing annotations are mostly for function signatures, hence the ambiguity :(